packages/e2e/detox/integration/common/shared.ts

```diff
@@ -1,17 +1,22 @@
 import { Given, Then, When } from '@cucumber/cucumber';
 import { by, element, expect } from 'detox';
 
+/**
+ * Use this util for replacing text instead of typing into input fields for faster test run duration
+ * Note: validations trigger after text was replaced, if you need to mimic user behavior use typeText instead
+ *
+ * @param inputFieldMatcher
+ * @param text
+ */
 export const typeInInputField = async (
   inputFieldMatcher: Detox.NativeMatcher,
   text: string
 ) => {
   const inputField = element(inputFieldMatcher);
-  await device.disableSynchronization();
```

Comment:
Seeing some warning in logs that could be related to this, give it a try without.

Reply:
Doesn't move the needle in terms of test duration, was required prior to Detox 20, not needed anymore.